api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -298,12 +296,11 @@ public void testInvitationKeyVerification_invitationKeyMismatch() {
 
   @Test(expected = BadRequestException.class)
   public void testCreateAccount_MismatchEmailAddress() {
-    createUser();
-    config.featureFlags.requireInstitutionalVerification = true;
     final Institution broad =
         new Institution()
             .shortName("Broad")
             .displayName("The Broad Institute")
+            .organizationTypeEnum(OrganizationType.ACADEMIC_RESEARCH_INSTITUTION)
```

Comment:
fun fact! many of these tests were invalid because leaving out the `organizationTypeEnum` when creating an Institution *also* generates a `BadRequestException`